### ISSUE TITLE
Web interface feedback + improvements

### ISSUE THREAD

**Issue by dginev** [dginev/CorTeX#22]
Need to address the items reported during the original user tests of the mojolicious-based interface and ensure parity and beyond with the new Rocket web service.

https://github.com/dginev/deprecated-CorTeX/issues/30

There are also new items to revisit, such as 
 * [x] ensuring individual downloads work
 * [x] adding pagination in the file report pages,
 * [x] making sure TeX.js is back to fully operational for previews.
 * [x] fixing figure previews in Firefox
 * [x] adding a multi-severity task list for a the leaf report pages (e.g. for a given warning, show a list of all Fatal severity tasks which had that warning)
 * [x] adding back the "log" download for Fatals, where you can quickly see the conversion log for a job
 * [x] adding a loading spinner when downloading ZIP files, it's very opaque right now
 * [x] switching to a dedicated preview page for Michael, which doesn't help debugging, but helps showcasing

**Comment by dginev** [comment]
I have added a dedicated preview page with an operational TeX.js installation today, and also included the conversion logs at the bottom of each article previewed.

This achieves most of the remaining items in this list. I also added the "all_tasks" view in the main report pages some time ago which is a close approximation to the "multi-severity" report requested here.

The two remaining items are:
 * pagination to allow access to all tasks in each category
 * loading spinner during preview page load. I think this is less needed with the dedicated page, as the page itself is blank until the article load succeeds.
